### ISSUE TITLE
Removing last row incorrect behavior

### ISSUE THREAD

**Issue by TimGa** [PHPOffice/PhpSpreadsheet#1364]
This is:

```
- [X] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
```
Before
+-----+                         After
|A1|B1|                         +-----+
+-----+                         |A1|B1|
|A2|B2| -> Removing last row -> +-----+
+-----+                         |A2|B2|
|A3|B3|                         +-----+
+-----+

After removing last 3rd row from the table
rest rows (1st and,2nd) should remain
```

### What is the current behavior?
```
Before
+-----+
|A1|B1|                         After
+-----+                         +-----+
|A2|B2| -> Removing last row -> |A1|B1|
+-----+                         +-----+
|A3|B3|
+-----+

But in fact when removing last 3rd row,
the 2nd row also disappears
```

### What are the steps to reproduce?
```php
<?php

require_once 'vendor/autoload.php';

use PhpOffice\PhpSpreadsheet\Spreadsheet;

$arrayData = [
    ['A1'],
    ['A2'],
    ['A3'],
];

$spreadsheet = new Spreadsheet();
$worksheet = $spreadsheet->getActiveSheet();
$worksheet->fromArray($arrayData);

$worksheet->removeRow(3, 1);

echo $worksheet->getCellByColumnAndRow(1, 1)->getValue() . PHP_EOL; // expect 'A1'
echo $worksheet->getCellByColumnAndRow(1, 2)->getValue() . PHP_EOL; // expect 'A2'
```
expected result is:
```
A1
A2
```
actual result is:
```
A1
```

### Which versions of PhpSpreadsheet and PHP are affected?
PhpSpreadsheet v1.10.1
PHP v7.4.2

**Comment by TimGa** [comment]
#1365

**Comment by oleibman** [comment]
Fixed by PR #1365.
